packages/gatsby/package.json

```diff
@@ -158,7 +160,7 @@
     "webpack": "^5.16.0",
     "webpack-dev-middleware": "^4.1.0",
     "webpack-dev-server": "^3.11.2",
-    "webpack-hot-middleware": "^2.25.0",
+    "webpack-hot-middleware": "npm:@gatsbyjs/webpack-hot-middleware@^2.25.2",
```

Comment:
will this work on every package manager? is there reason to use alias and not specify `"@gatsbyjs/webpack-hot-middleware": "^2.25.2"` and adjust imports?

Reply:
We need to fix this import here, too:
https://github.com/pmmmwh/react-refresh-webpack-plugin/blob/main/sockets/WHMEventSource.js#L14

Reply:
that seems like client code so we could alias it with webpack rules - I just worry that some package managers might not support aliases. I.e. `npm` added support in 6.9 and node12.13 seems to come with `npm@6.12` so this should be good, but I'm not sure about other package managers

Reply:
I have a "@gatsbyjs/react-refresh-webpack-plugin" let me wire it in

Reply:
done in https://github.com/gatsbyjs/gatsby/pull/29588/commits/b8c469b03915ecc23acca36bf18016306d0b4765